src/ticker_normalization.c

```diff
@@ -0,0 +1,71 @@
+#include <cx.h>
```

Comment:
Is this import really needed? Couldn't it be replaced with 'standard' import such as `<string.h>`?
My though is that at some point it would be great to have C unit testing, and this module seems a good entrypoint. `string.h` is free basically, whereas `cx.h` would require some mocking.

Reply:
Removed